content/spaces/helm-reference.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Helm Chart Reference
+weight: 200
+description: Spaces Helm chart configuration values
+---
+
+This reference provides detailed documentation on the Upbound Space Helm chart. This page details the Helm chart configuration values for installation, configuration, and management of an Upbound Space deployment.
+
+## Top-level keys
+
+[account](#account)
+
+[clusterType](#clustertype)
+
+[version](#version)
+
+[ingress](#ingress)
+
+[deletion policy](#deletion-policy)
+
+[space](#space)
+
+[certificates](#certificates)
+
+[router](#router)
+
+[billing](#billing)
+
+[features](#features)
+
+
+### account
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| account | string | `"notdemo"` | The Upbound organization this installation is associated with. |
+{{</ table >}}
+
+### clusterType
+
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| clusterType | string | `"kind"` | Specifies the cluster type that this installation is being installed into. <br> Valid options are: `aks`, `eks`, `gke`, `kind`. |
+{{</ table >}}
+
+### version
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| version | string | `"0.1.0"` | Overall artifact version that affects xpkgs and related components. |
+{{</ table >}}
+
+### ingress
+
+{{< table >}}
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| ingress | object | | Configurations for external requests coming into the space. |
```

Comment:
I wouldn't include keys that don't have configurable settings. Leave it up to the child keys like `ingress.host`